apps/client-server/src/app/post-parsers/models/description-node.spec.ts

```diff
@@ -51,7 +51,7 @@ describe('DescriptionNode', () => {
     );
 
     expect(tree.toPlainText()).toBe('Hello, https://test.postybirb.com/User');
-    expect(tree.toHTML()).toBe(
+    expect(tree.toHtml()).toBe(
```

Comment:
These all were type errors reported by ci so i had to fix them. No actual tests are working right now